doc/dialects/postgresql.md

```diff
@@ -1,3 +1,3 @@
 # PostgreSQL SQL Dialect User Guide
 
-The PostgreSQL dialect user guide can be found at: https://github.com/exasol/postgresql-virtual-schema/blob/main/doc/user_guide/postgresql_user_guide.md
+The PostgreSQL dialect user guide can be found at: https://github.com/exasol/postgresql-virtual-schema/blob/main/doc/user_guide/postgresql_user_guide.md.
```

Comment:
This seems kind of self-referencing. Why does that line exist?

Reply:
Yes, I agree. 
I found references from other places pointing to this file.
* https://github.com/exasol/virtual-schemas/blob/main/README.md
Maybe this file is intended to provide some kind of legacy compatibility in form of not breaking links.
Should we better delete the file?

Reply:
Removed the file